packages/fxa-content-server/app/scripts/models/auth_brokers/oauth-redirect.js

```diff
@@ -99,8 +99,8 @@ export default BaseAuthenticationBroker.extend({
    * @returns {Promise}
    */
   sendOAuthResultToRelier(result) {
-    return Promise.resolve().then(() => {
-      var extraParams = {};
+    return this._metrics.flush().then(() => {
```

Comment:
This method was being called outside of `finishOAuthFlow` where metrics were originally flushed. Now flushing here to ensure all metrics are sent before redirecting.